submissions/another96/html-css-popup/index.html

```diff
@@ -0,0 +1,145 @@
+                                 class="icon">
+                              <p class="popup-text">Work</p>
+                           </a>
+                        </li>
+                        <li tabindex="0" class="sub-nav-item">
+                           <a href="" class="popup-link" tabindex="-1">
+                              <img src="img/info.png" alt=""
+                                 class="icon">
+                              <p class="popup-text">Info</p>
+                           </a>
+                        </li>
+                     </ul>
+                     <input type="checkbox" id="sub-popup-input" class="second-input">
+                     <label for="sub-popup-input" class="sub-popup-label">
+                        <p class="subpopup-text">Social Media</p>
+                     </label>
+                     <div class="sub-popup-container">
+                        <ul class="sub-popup">
+                           <li tabindex="0" class="sub-nav-item">
+                              <a href="" class="popup-link" tabindex="-1">
+                                 <img src="img/instagram.png"
+                                    alt="" class="icon">
+                                 <p class="popup-text">Instagram</p>
+                              </a>
+                           </li>
+                           <li tabindex="0" class="sub-nav-item">
+                              <a href="" class="popup-link" tabindex="-1">
+                                 <img src="img/twitter.png"
+                                    alt="" class="icon">
+                                 <p class="popup-text">twitter</p>
+                              </a>
+                           </li>
+                           <li tabindex="0" class="sub-nav-item">
+                              <a href="" class="popup-link" tabindex="-1">
+                                 <img
+                                    src="img/facebook-free-icon-font.png" alt="" class="icon">
+                                 <p class="popup-text">Facebook</p>
+                              </a>
+                           </li>
+                           <li tabindex="0" class="sub-nav-item">
+                              <a href="" class="popup-link" tabindex="-1">
+                                 <img src="img/whatsapp.png"
+                                    alt="" class="icon">
+                                 <p class="popup-text">Whatsapp</p>
+                              </a>
+                           </li>
+                           <li tabindex="0" class="sub-nav-item">
+                              <a href="" class="popup-link" tabindex="-1">
+                                 <img src="img/linkedin.png"
+                                    alt="" class="icon">
+                                 <p class="popup-text">Linkedin</p>
+                              </a>
+                           </li>
```

Comment:
By default, the browser focuses on interactive elements, so it makes no sense to set the `tabindex` attribute on list elements and at the same time hide the interactive element itself. Only if you are not performing some specific task